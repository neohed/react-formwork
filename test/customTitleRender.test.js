import React from 'react';
import CustomTitleForm from './forms/customTitleForm';
import renderer from 'react-test-renderer';

test('Test a form with custom form field titles', () => {
    const component = renderer.create(
        <CustomTitleForm />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
