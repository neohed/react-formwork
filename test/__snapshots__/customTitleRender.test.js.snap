// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test a form with custom form field titles 1`] = `
<div>
  <div>
    <label
      htmlFor="name"
    >
      First Name
    </label>
    <input
      name="name"
      onBlur={[Function]}
      onChange={[Function]}
      type="text"
      value=""
    />
    
  </div>
  <div>
    <label
      htmlFor="email"
    >
      Primary Email Address
    </label>
    <input
      name="email"
      onBlur={[Function]}
      onChange={[Function]}
      type="text"
      value=""
    />
    
  </div>
</div>
`;
